Steamworks/src/org/usfirst/frc/team2363/robot/subsystems/Pixy.java

```diff
@@ -0,0 +1,98 @@
+package org.usfirst.frc.team2363.robot.subsystems;
+
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import static org.usfirst.frc.team2363.robot.Robot.pixy;
+
+import org.usfirst.frc.team2363.robot.commands.PixyCam.*;
+
+/**
+ *
+ */
+public class Pixy extends Subsystem {
+	
+	// declares the different Pixy Cam communication states
+    public enum PixyState {
+    	ON,
+    	OFF
+    }
+    public static I2C pixyi2c;
+	public PixyState state;
+
+	public Pixy() {
+		
+		pixyi2c = new I2C(I2C.Port.kOnboard, 0x54);
+		state = PixyState.OFF;
+	}
+	
+    public void initDefaultCommand() {
+        // Set the default command for a subsystem here.
+    	setDefaultCommand(new NoTargetAcquire());
+    }
+
+	public void on() {
```

Comment:
I don't know that we really need to keep track of the state of the pixy. We should just be able to get away with a read method that reads the state of the pixy and returns the desired x position we are looking for.